decentraland/unity-renderer
# feat: Be able to join a channel using a deep link

## What does this PR change?
In order to attract communities from other social networks, we need to be able to create links that can be posted anywhere, and people can click, so they automatically join the channel.

Independently, this will prompt the confirmation dialog to join the channel in-game.

This will be done using a url param like `&CHANNEL=name-of-channel`

![image](https://user-images.githubusercontent.com/64659061/202285917-c4ecd638-144a-46d4-87a1-15e446f538db.png)

### Acceptance Criteria
- [x] Join Channel modal must be automatically open when the `CHANNEL` param is declared in the URL.

### Important notes
- You will need to have permissions to create channels (it is configured in the `users_allowed_to_create_channels` feature flag's variant) in order to see this feature.

### Dependencies
This PR depends on this kernel PR: https://github.com/decentraland/kernel/pull/710

## How to test the changes?
1. Go to: https://play.decentraland.org/?renderer-branch=feat/be-able-to-join-a-channel-using-a-deep-link&CHANNEL=global
2. Notice the Join Channel modal appears just after Channels feature is initialized.

## Our Code Review Standards
https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
